PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp

```diff
@@ -2589,6 +2589,91 @@ void ScintillaEditView::currentLinesDown() const
 	execute(SCI_SCROLLRANGE, execute(SCI_GETSELECTIONEND), execute(SCI_GETSELECTIONSTART));
 }
 
+void ScintillaEditView::multiCursorLeftOrRight(int direction, bool shouldExt) const
+{
+    // Get the number of selection
+    int numSelections = int(execute(SCI_GETSELECTIONS));
+
+    // Use default functions when there is one selection
+    if (numSelections == 1) {

```

Comment:
bad coding style in the added codes:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#general